### PR TITLE
Adopting code quality parent pom

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,12 +78,19 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
-            <version>1.5.10.RELEASE</version>
+            <version>${spring.boot.version}</version>
```
Comment (cschroed-usgs) [inline pom.xml]: Not sure, but I think the starter-test version and main framework version are supposed to agree. Thoughts?

Comment (zmoore-usgs) [reply pom.xml]: Yep, thanks for catching that

**Comment by cschroed-usgs** [comment]
Thanks Zack.

Sorry, I initially proposed adopting an old version of the plugin and consequently a configuration that invited build instability.

The potential instability arises from the security scan's use of NIST's severely rate-limited National Vulnerability Database. After hitting it x times in one day, it denies your requests, which would fail your build.

To ensure build stability, we can either: point our builds at a robust mirror of the NVD or shut off the scan. The latest stable code quality parent pom points at the internal USGS NVD mirror by default, but since the mirror is only available on the USGS network, that's not an option for Travis builds. Fortunately, we can still use this in the project's Jenkins build.

Ok. ALL that said, still mergeworthy?

**Comment by cschroed-usgs** [comment]
Huh. I saw that this failed. Gotta dash unfortunately. Not sure why it's running a `mvn install` command in addition to the `mvn clean verify` one that I updated. I will dig in tomorrow.

**Comment by cschroed-usgs** [comment]
Turns out Travis runs a questionable mvn command by default during its install phase.
https://docs.travis-ci.com/user/languages/java/#maven-dependency-management

Seems like Travis engineers set up that default without really understanding what the `mvn install` command does. Fortunately there's an easy opt-out that I included in this PR.

https://stackoverflow.com/questions/31945809/skip-first-mvn-install-in-travis-ci

**Pull request merged**